README.md

```diff
@@ -113,6 +113,7 @@ Ruby:
 - mri
 - jruby
 - rubocop
+- rubylint
 - reek
```

Comment:
I think the order should reflect the actual one from `neomake#makers#ft#ruby#EnabledMakers`.